types/stripe/index.d.ts

```diff
@@ -9421,6 +9746,41 @@ declare namespace Stripe {
                 redirectUrl?: string,
                 response?: IResponseFn<accounts.ILoginLink>,
             ): Promise<accounts.ILoginLink>;
+
```

Comment:
I believe all of these CRUDL functions should have an overload that accepts a `HeaderOptions` param before the `IResponseFn`.

Reply:
Good catch, I didn't think to look at the other CRUDL types and only looked at the documentation where this isn't explicitly mentioned on each function. I _think_ it should be correct now